src/plugin/proxy/utils.ts

```diff
@@ -120,7 +121,8 @@ export const generateRoutes = (contents: Array<YAMLRoute> = []) => {
           iam: {
             permissions: route?.permissions || [],
             hooks: route?.hooks || []
-          }
+          },
+          jsonpath: route?.jsonpath
```

Comment:
can we use some other variable name as it gives the impression that it supports the `jsonpath` syntax and parsing?
Either change the variable name or implement [jsonpath](https://jsonpath.com)